lms/djangoapps/verify_student/views.py

```diff
@@ -1154,10 +1154,22 @@ def results_callback(request):
     except SoftwareSecurePhotoVerification.DoesNotExist:
         log.error("Software Secure posted back for receipt_id %s, but not found", receipt_id)
         return HttpResponseBadRequest("edX ID {} not found".format(receipt_id))
+    context = {'user': attempt.user}
+    email_template_context = {'platform_name': settings.PLATFORM_NAME}
     if result == "PASS":
         log.debug("Approving verification for %s", receipt_id)
         attempt.approve()
         status = "approved"
+        expiry_date = datetime.date.today() + datetime.timedelta(days=365)
```

Comment:
shouldn't we be using settings.VERIFY_STUDENT["DAYS_GOOD_FOR"] to calculate the expiration date?